pkg/commonvpc/vpc.go

```diff
@@ -0,0 +1,333 @@
+/*
```

Comment:
Same question here - what is really new? why is the entire file identified as new?